tests/EdgeToEdge/Routes/AddDonationRouteTest.php

```diff
@@ -357,9 +358,10 @@ public function testGivenValidCreditCardData_showsIframeEmbeddingPage(): void {
 		);
 
 		$response = $client->getResponse();
-		$this->assertSame( 200, $response->getStatusCode() );
-		$this->assertContains( 'paytext_cc 3 12,34 € per Kreditkarte.', $response->getContent() );
-		$this->assertContains( 'thatother.paymentprovider.com', $response->getContent() );
+		$this->assertSame( Response::HTTP_FOUND, $response->getStatusCode() );
+		$this->assertTrue( $response->isRedirect() );
+		$this->assertContains( 'amount=1234', $response->getContent() );
```

Comment:
I believe this is some fancy pants behaviour of symfony http response which adds a response body (which is not strictly forbidden as per RFC) with a link, maybe meta redirect, for clients not able/allowed to follow location headers... I anyway think we should properly assert the header, not the content.